CHANGELOG.rst

```diff
@@ -6,6 +6,291 @@
 Synapse Changelog
 *****************
 
+v2.165.0 - TBD
+=====================
+
+Automatic Migrations
+--------------------
+- Re-normalize ``risk:mitigation:name``, ``it:mitre:attack:technique:name``,
+ and ``it:mitre:attack:mitigation:name`` secondary properties.
+  (`#3585 <https://github.com/vertexproject/synapse/pull/3585>`_)
+- Re-normalize ``velocity`` properties which are float values.
+  (`#3616 <https://github.com/vertexproject/synapse/pull/3616>`_)
+- See :ref:`datamigration` for more information about automatic migrations.
+
+Model Changes
+-------------
+- Add a new model, ``sci``, for modeling scientific experiments. Updates to
+  the ``econ``, ``file``, ``infotech``, ``inet``, ``ou``, ``ps``, and ``risk``
+  models.
+  (`#3559 <https://github.com/vertexproject/synapse/pull/3559>`_)
+  (`#3585 <https://github.com/vertexproject/synapse/pull/3585>`_)
+  (`#3595 <https://github.com/vertexproject/synapse/pull/3595>`_)
+  (`#3604 <https://github.com/vertexproject/synapse/pull/3604>`_)
+  (`#3606 <https://github.com/vertexproject/synapse/pull/3606>`_)
+  (`#3622 <https://github.com/vertexproject/synapse/pull/3622>`_)
+
+  **New Forms**
+
+  ``econ:acct:receipt``
+    A receipt issued as proof of payment.
+
+  ``econ:acct:invoice``
+    An invoice issued requesting payment.
+
+  ``econ:bank:account:type:taxonomy``
+    A bank account type taxonomy.
+
+  ``econ:bank:account``
+    A bank account.
+
+  ``econ:bank:balance``
+    A balance contained by a bank account at a point in time.
+
+  ``econ:bank:statement``
+    TODO FIXME
+
+  ``econ:bank:aba:rtn``
+    An American Bank Association (ABA) routing transit number (RTN).
+
+  ``econ:bank:iban``
+    An International Bank Account Number.
+
+  ``econ:bank:swift:bic``
+    A Society for Worldwide Interbank Financial Telecommunication (SWIFT)
+    Business Identifier Code (BIC).
+
+  ``risk:vulnerable``
+    Indicates that a node is susceptible to a vulnerability.
+
+  ``sci:hypothesis:type:taxonomy``
+    FIXME MISSING DOCSTRING
```

Comment:
Need to smoke in some doc strings for these two FIXMEs

Reply:
Yup - already done.